app/src/main/java/grinnell/appdev/edu/lyles/fragments/BeerFragment.java

```diff
@@ -1,22 +1,127 @@
 package grinnell.appdev.edu.lyles.fragments;
 
+import android.os.AsyncTask;
 import android.os.Bundle;
-import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
 import grinnell.appdev.edu.lyles.R;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class BeerFragment extends Fragment{
 
-/**
- * Created by Mattori on 5/9/16.
- */
-public class BeerFragment extends Fragment {
+    ArrayList<Beers> beersList;
```

Comment:
You should make these member variables private and use the 'mVariableName' naming convention